File: eCar/UI/ecar_admin/lib/models/Notification/notification.dart
import 'package:json_annotation/json_annotation.dart';
part 'notification.g.dart';

@JsonSerializable()
class Notification {
  int? id;
  String? heading;
  String? content_;
  String? image;
  DateTime? addingDate;
  bool? isForClient;
  String? status;

  Notification({
    this.id,
    this.heading,
    this.content_,
    this.image,
    this.addingDate,
    this.isForClient,
    this.status,
  });

  factory Notification.fromJson(Map<String, dynamic> json) =>
      _$NotificationFromJson(json);

  Map<String, dynamic> toJson() => _$NotificationToJson(this);
}

File: eCar/eCar.Model/Model/Notification.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCar.Model.Model
{
    public  class Notification
    {
        public int Id { get; set; }

        public string? Heading { get; set; }

        public string? Content_ { get; set; }

        public byte[]? Image { get; set; }

        public DateTime? AddingDate { get; set; }

        public bool? IsForClient { get; set; }

        public string? Status { get; set; }
    }
}
